projects/goharbor/harbor/docker/linux/harbor-redis/Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG BASE_IMAGE
+ARG BUILDER_IMAGE
+FROM $BUILDER_IMAGE as redis-builder
```

Comment:
I don't want to hold this up because of this problem, but I want to hear what happened here.

Reply:
I followed the same steps that we built the existing redis image but replaced the configuration file as the existing redis image failed to work in Harbor eco-system.